gufe/tokenization.py

```diff
@@ -360,6 +360,15 @@ def __repr__(self):   # pragma: no cover
     def to_dict(self):
         return {':gufe-key:': str(self)}
 
+    @property
+    def prefix(self):
+        return self.split('-')[0]
+
+    @property
+    def token(self):
+        return self.split('-')[1]
```

Comment:
not bad, but tests

Prefix is always the classname though right?

Reply:
Yes, prefix is always the classname. Perhaps instead of these properties pulling apart the string representation, we hold onto these components separately by overriding `__init__`?

Reply:
:pray: Yes, much more clear than remembering what `[0]` and `[1]` are